server/statshouse/statshouse-manager.cpp

```diff
@@ -116,6 +118,9 @@ void StatsHouseManager::add_request_stats(uint64_t script_time_ns, uint64_t net_
 
   client.metric("kphp_requests_outgoing_queries").tag(worker_type).write_value(script_queries);
   client.metric("kphp_requests_outgoing_long_queries").tag(worker_type).write_value(long_script_queries);
+
+  client.metric("kphp_requests_scheduler_context_switches").tag("voluntary").tag(status).write_value(script_rusage.voluntary_context_switches);
+  client.metric("kphp_requests_scheduler_context_switches").tag("involuntary").tag(status).write_value(script_rusage.involuntary_context_switches);
```

Comment:
And also let's add `kphp_by_host_request_scheduler_context_switches` with tag host always enabled